.github/workflows/test.yml

```diff
@@ -40,6 +40,12 @@ jobs:
 
     # Static checks from this point forward. Only run on one Go version and on
     # Linux, since it's the fastest platform, and the tools behave the same.
+    - if: matrix.os == 'ubuntu-latest' && matrix.go-version == '1.18.x'
```

Comment:
Maybe it makes sense to test in windows as well?

Reply:
Unfortunately Windows on GitHub Actions is extremely slow. Just running `go test ./...` takes ~13m versus ~4m on Linux. Windows is already the bottleneck for CI, even when Linux does way more (race tests, staticcheck, third party tests, etc).

Do you mean building _on_ windows or _for_ windows? We could always build on Linux with `GOOS=windows GOARCH=arm` as a way to diversify the operating systems and architectures, and then we don't have to suffer from the slow Windows machines as much.

Reply:
I meant "building on windows" to make sure that the difference in systems does not cause a problems (as there was once a problem with symlink). But these are unlikely and I don't think critical problems and for the sake of it there is no point in slowing down the build so much

Reply:
Yeah, I think the standard tests already give us enough coverage for the host OS. It's a balance for sure - I think the main thing we want with third party packages is variety of code being built.